Develop -> UAT

SamThomasJ opened the pull request.

NToppingCCS committed:
Switch to the new AS.

This involves:
- Updating the headers to inculde an API key
- Update the healthcheck endpoint to use the API key
- Update the routes
- Make sure the tests continue to work

tim-s-ccs committed:
Update the PPG endpoints

tim-s-ccs committed:
Merge pull request #4387 from Crown-Commercial-Service/feature/CAS-1922-ppg-endpoint-change

CAS-1922 - Update PPG Endpoints

dependabot[bot] committed:
Bump msw from 1.3.0 to 1.3.2

Bumps [msw](https://github.com/mswjs/msw) from 1.3.0 to 1.3.2.
- [Release notes](https://github.com/mswjs/msw/releases)
- [Changelog](https://github.com/mswjs/msw/blob/main/CHANGELOG.md)
- [Commits](https://github.com/mswjs/msw/compare/v1.3.0...v1.3.2)

---
updated-dependencies:
- dependency-name: msw
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

SamThomasJ committed:
CAS-1942 Remove GCloud Customer Guidance Link From Related Content

dependabot[bot] committed:
Merge pull request #4412 from Crown-Commercial-Service/dependabot/npm_and_yarn/msw-1.3.2

dependabot[bot] committed:
Bump @babel/core from 7.22.17 to 7.23.0

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.22.17 to 7.23.0.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.23.0/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4411 from Crown-Commercial-Service/dependabot/npm_and_yarn/babel/core-7.23.0

dependabot[bot] committed:
Bump @types/jsdom from 21.1.2 to 21.1.3

Bumps [@types/jsdom](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/jsdom) from 21.1.2 to 21.1.3.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/jsdom)

---
updated-dependencies:
- dependency-name: "@types/jsdom"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4396 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/jsdom-21.1.3

SamThomasJ committed:
Merge branch 'develop' into CAS-1942-remove-link-throughout-g-cloud-user-journey-in-cas

SamThomasJ committed:
CAS-1982 Fix Broken Link on Supplier List Pages When Not Signed In

SamThomasJ committed:
CAS-2000 Remove grey bar from top of chrome

SamThomasJ committed:
Merge pull request #4415 from Crown-Commercial-Service/CAS-1942-remove-link-throughout-g-cloud-user-journey-in-cas

CAS-1942 Remove GCloud Customer Guidance Link From Related Content

SamThomasJ committed:
Merge branch 'develop' into CAS-1982-broken-lot-supplier-details-pages

SamThomasJ committed:
Merge pull request #4420 from Crown-Commercial-Service/CAS-1982-broken-lot-supplier-details-pages

CAS-1982 Fix Broken Link on Supplier List Pages When Not Signed In

SamThomasJ committed:
Merge branch 'develop' into CAS-2000-remove-grey-bar-at-top-of-page-only-appears-on-chrome

SamThomasJ committed:
Merge pull request #4421 from Crown-Commercial-Service/CAS-2000-remove-grey-bar-at-top-of-page-only-appears-on-chrome

CAS-2000 Remove grey bar from top of chrome

NToppingCCS commented:
I think we possibly do need to hold off on this after all @SamThomasJ @tim-s-ccs because we do have endpoint change config in there too - so if the PPG environment UAT points to hasn't been moved to AWS yet, this would break it.

Annoyingly, may have to keep this paused for now.

tim-s-ccs committed:
Target a node version of 20 so that we can always use the latest build pack on GPaaS

Change the version of Mock Service Worker we are using and update the tests to accommodate the changes.

tim-s-ccs committed:
Merge pull request #4426 from Crown-Commercial-Service/reduce-node-version-restrictions

Target a node version of 20 so that we can always use the latest buil…

dependabot[bot] committed:
Bump @babel/traverse from 7.23.0 to 7.23.2

Bumps [@babel/traverse](https://github.com/babel/babel/tree/HEAD/packages/babel-traverse) from 7.23.0 to 7.23.2.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.23.2/packages/babel-traverse)

---
updated-dependencies:
- dependency-name: "@babel/traverse"
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

tim-s-ccs committed:
Merge pull request #4425 from Crown-Commercial-Service/dependabot/npm_and_yarn/babel/traverse-7.23.2

Bump @babel/traverse from 7.23.0 to 7.23.2

dependabot[bot] committed:
Bump jquery from 3.7.0 to 3.7.1

Bumps [jquery](https://github.com/jquery/jquery) from 3.7.0 to 3.7.1.
- [Release notes](https://github.com/jquery/jquery/releases)
- [Commits](https://github.com/jquery/jquery/compare/3.7.0...3.7.1)

---
updated-dependencies:
- dependency-name: jquery
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4419 from Crown-Commercial-Service/dependabot/npm_and_yarn/jquery-3.7.1

dependabot[bot] committed:
Bump @hmcts/nodejs-healthcheck from 1.8.0 to 1.8.3

Bumps [@hmcts/nodejs-healthcheck](https://github.com/hmcts/nodejs-healthcheck) from 1.8.0 to 1.8.3.
- [Release notes](https://github.com/hmcts/nodejs-healthcheck/releases)
- [Commits](https://github.com/hmcts/nodejs-healthcheck/compare/1.8.0...1.8.3)

---
updated-dependencies:
- dependency-name: "@hmcts/nodejs-healthcheck"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4397 from Crown-Commercial-Service/dependabot/npm_and_yarn/hmcts/nodejs-healthcheck-1.8.3

dependabot[bot] committed:
Bump i18next from 23.4.4 to 23.6.0

Bumps [i18next](https://github.com/i18next/i18next) from 23.4.4 to 23.6.0.
- [Release notes](https://github.com/i18next/i18next/releases)
- [Changelog](https://github.com/i18next/i18next/blob/master/CHANGELOG.md)
- [Commits](https://github.com/i18next/i18next/compare/v23.4.4...v23.6.0)

---
updated-dependencies:
- dependency-name: i18next
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/cookies from 0.7.7 to 0.7.9

Bumps [@types/cookies](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/cookies) from 0.7.7 to 0.7.9.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/cookies)

---
updated-dependencies:
- dependency-name: "@types/cookies"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump del from 7.0.0 to 7.1.0

Bumps [del](https://github.com/sindresorhus/del) from 7.0.0 to 7.1.0.
- [Release notes](https://github.com/sindresorhus/del/releases)
- [Commits](https://github.com/sindresorhus/del/compare/v7.0.0...v7.1.0)

---
updated-dependencies:
- dependency-name: del
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4431 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/cookies-0.7.9

dependabot[bot] committed:
Bump @types/config from 3.3.0 to 3.3.2

Bumps [@types/config](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/config) from 3.3.0 to 3.3.2.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/config)

---
updated-dependencies:
- dependency-name: "@types/config"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

tim-s-ccs committed:
Remove the package as it is not used

dependabot[bot] committed:
Merge pull request #4433 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/config-3.3.2

dependabot[bot] committed:
Bump @types/sinon-chai from 3.2.9 to 3.2.11

Bumps [@types/sinon-chai](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/sinon-chai) from 3.2.9 to 3.2.11.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/sinon-chai)

---
updated-dependencies:
- dependency-name: "@types/sinon-chai"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4428 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/sinon-chai-3.2.11

dependabot[bot] committed:
Bump @types/require-directory from 2.1.2 to 2.1.4

Bumps [@types/require-directory](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/require-directory) from 2.1.2 to 2.1.4.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/require-directory)

---
updated-dependencies:
- dependency-name: "@types/require-directory"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4429 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/require-directory-2.1.4

dependabot[bot] committed:
Bump @types/node from 20.6.0 to 20.8.7

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 20.6.0 to 20.8.7.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4427 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/node-20.8.7

dependabot[bot] committed:
Merge pull request #4395 from Crown-Commercial-Service/dependabot/npm_and_yarn/del-7.1.0

tim-s-ccs committed:
Merge pull request #4432 from Crown-Commercial-Service/dependabot/npm_and_yarn/i18next-23.6.0

Bump i18next from 23.4.4 to 23.6.0

dependabot[bot] committed:
Bump eslint from 8.49.0 to 8.52.0

Bumps [eslint](https://github.com/eslint/eslint) from 8.49.0 to 8.52.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v8.49.0...v8.52.0)

---
updated-dependencies:
- dependency-name: eslint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4440 from Crown-Commercial-Service/dependabot/npm_and_yarn/eslint-8.52.0

dependabot[bot] committed:
Bump @types/cookie-parser from 1.4.3 to 1.4.5

Bumps [@types/cookie-parser](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/cookie-parser) from 1.4.3 to 1.4.5.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/cookie-parser)

---
updated-dependencies:
- dependency-name: "@types/cookie-parser"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4442 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/cookie-parser-1.4.5

dependabot[bot] committed:
Bump @types/serve-favicon from 2.5.4 to 2.5.6

Bumps [@types/serve-favicon](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/serve-favicon) from 2.5.4 to 2.5.6.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/serve-favicon)

---
updated-dependencies:
- dependency-name: "@types/serve-favicon"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4443 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/serve-favicon-2.5.6

dependabot[bot] committed:
Bump msw from 0.0.0-fetch.rc-24 to 2.0.1

Bumps [msw](https://github.com/mswjs/msw) from 0.0.0-fetch.rc-24 to 2.0.1.
- [Release notes](https://github.com/mswjs/msw/releases)
- [Changelog](https://github.com/mswjs/msw/blob/main/CHANGELOG.md)
- [Commits](https://github.com/mswjs/msw/commits/v2.0.1)

---
updated-dependencies:
- dependency-name: msw
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4441 from Crown-Commercial-Service/dependabot/npm_and_yarn/msw-2.0.1

dependabot[bot] committed:
Bump @types/nunjucks from 3.2.3 to 3.2.5

Bumps [@types/nunjucks](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/nunjucks) from 3.2.3 to 3.2.5.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/nunjucks)

---
updated-dependencies:
- dependency-name: "@types/nunjucks"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4444 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/nunjucks-3.2.5

dependabot[bot] committed:
Bump @babel/core from 7.23.0 to 7.23.2

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.23.0 to 7.23.2.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/@babel/core@7.23.2/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4446 from Crown-Commercial-Service/dependabot/npm_and_yarn/babel/core-7.23.2

dependabot[bot] committed:
Bump @types/express from 4.17.17 to 4.17.20

Bumps [@types/express](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/express) from 4.17.17 to 4.17.20.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/express)

---
updated-dependencies:
- dependency-name: "@types/express"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4447 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/express-4.17.20

dependabot[bot] committed:
Bump @types/pa11y from 5.3.5 to 5.3.6

Bumps [@types/pa11y](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/pa11y) from 5.3.5 to 5.3.6.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/pa11y)

---
updated-dependencies:
- dependency-name: "@types/pa11y"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4448 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/pa11y-5.3.6

dependabot[bot] committed:
Bump @types/csurf from 1.11.2 to 1.11.4

Bumps [@types/csurf](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/csurf) from 1.11.2 to 1.11.4.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/csurf)

---
updated-dependencies:
- dependency-name: "@types/csurf"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4445 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/csurf-1.11.4

dependabot[bot] committed:
Bump @typescript-eslint/eslint-plugin from 5.60.1 to 6.9.1

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 5.60.1 to 6.9.1.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v6.9.1/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

tim-s-ccs committed:
Update config for major change to the typescript plugin.

I've updated the rules we used and turned some off as they would require far to many changes to fix.
I have, however, autofixed anything new that eslint could autofix.

tim-s-ccs committed:
Merge pull request #4449 from Crown-Commercial-Service/dependabot/npm_and_yarn/typescript-eslint/eslint-plugin-6.9.1

Bump @typescript-eslint/eslint-plugin from 5.60.1 to 6.9.1

SamThomasJ committed:
CAS-1944 Push Cookie Preferences to DataLayer When Changing Values

SamThomasJ committed:
Merge pull request #4455 from Crown-Commercial-Service/CAS-1944-enable-data-layer-for-google-tag-manager-cookies-consent-management-part-1

CAS-1944 Push Cookie Preferences to DataLayer When Changing Values

dependabot[bot] committed:
Bump sinon and @types/sinon

Bumps [sinon](https://github.com/sinonjs/sinon) and [@types/sinon](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/sinon). These dependencies needed to be updated together.

Updates `sinon` from 15.2.0 to 17.0.1
- [Release notes](https://github.com/sinonjs/sinon/releases)
- [Changelog](https://github.com/sinonjs/sinon/blob/main/docs/changelog.md)
- [Commits](https://github.com/sinonjs/sinon/compare/v15.2.0...v17.0.1)

Updates `@types/sinon` from 10.0.16 to 17.0.0
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/sinon)

---
updated-dependencies:
- dependency-name: sinon
  dependency-type: direct:development
  update-type: version-update:semver-major
- dependency-name: "@types/sinon"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4459 from Crown-Commercial-Service/dependabot/npm_and_yarn/sinon-and-types/sinon-17.0.1

dependabot[bot] committed:
Bump chai from 4.3.8 to 4.3.10

Bumps [chai](https://github.com/chaijs/chai) from 4.3.8 to 4.3.10.
- [Release notes](https://github.com/chaijs/chai/releases)
- [Changelog](https://github.com/chaijs/chai/blob/4.x.x/History.md)
- [Commits](https://github.com/chaijs/chai/compare/v4.3.8...v4.3.10)

---
updated-dependencies:
- dependency-name: chai
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4460 from Crown-Commercial-Service/dependabot/npm_and_yarn/chai-4.3.10

dependabot[bot] committed:
Bump eslint from 8.52.0 to 8.53.0

Bumps [eslint](https://github.com/eslint/eslint) from 8.52.0 to 8.53.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v8.52.0...v8.53.0)

---
updated-dependencies:
- dependency-name: eslint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4461 from Crown-Commercial-Service/dependabot/npm_and_yarn/eslint-8.53.0

dependabot[bot] committed:
Bump prettier from 3.0.2 to 3.0.3

Bumps [prettier](https://github.com/prettier/prettier) from 3.0.2 to 3.0.3.
- [Release notes](https://github.com/prettier/prettier/releases)
- [Changelog](https://github.com/prettier/prettier/blob/main/CHANGELOG.md)
- [Commits](https://github.com/prettier/prettier/compare/3.0.2...3.0.3)

---
updated-dependencies:
- dependency-name: prettier
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4465 from Crown-Commercial-Service/dependabot/npm_and_yarn/prettier-3.0.3

dependabot[bot] committed:
Bump redis from 4.6.8 to 4.6.10

Bumps [redis](https://github.com/redis/node-redis) from 4.6.8 to 4.6.10.
- [Release notes](https://github.com/redis/node-redis/releases)
- [Changelog](https://github.com/redis/node-redis/blob/master/CHANGELOG.md)
- [Commits](https://github.com/redis/node-redis/compare/redis@4.6.8...redis@4.6.10)

---
updated-dependencies:
- dependency-name: redis
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4463 from Crown-Commercial-Service/dependabot/npm_and_yarn/redis-4.6.10

dependabot[bot] committed:
Bump @types/node from 20.8.7 to 20.9.0

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 20.8.7 to 20.9.0.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4468 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/node-20.9.0

dependabot[bot] committed:
Bump @types/mocha from 10.0.1 to 10.0.4

Bumps [@types/mocha](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/mocha) from 10.0.1 to 10.0.4.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/mocha)

---
updated-dependencies:
- dependency-name: "@types/mocha"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4467 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/mocha-10.0.4

dependabot[bot] committed:
Bump @types/supertest from 2.0.12 to 2.0.16

Bumps [@types/supertest](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/supertest) from 2.0.12 to 2.0.16.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/supertest)

---
updated-dependencies:
- dependency-name: "@types/supertest"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4466 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/supertest-2.0.16

dependabot[bot] committed:
Bump @babel/preset-env from 7.22.10 to 7.23.3

Bumps [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) from 7.22.10 to 7.23.3.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.23.3/packages/babel-preset-env)

---
updated-dependencies:
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4469 from Crown-Commercial-Service/dependabot/npm_and_yarn/babel/preset-env-7.23.3

dependabot[bot] committed:
Bump @types/es6-promisify from 6.0.1 to 6.0.4

Bumps [@types/es6-promisify](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/es6-promisify) from 6.0.1 to 6.0.4.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/es6-promisify)

---
updated-dependencies:
- dependency-name: "@types/es6-promisify"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4470 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/es6-promisify-6.0.4

dependabot[bot] committed:
Bump @types/jsdom from 21.1.3 to 21.1.5

Bumps [@types/jsdom](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/jsdom) from 21.1.3 to 21.1.5.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/jsdom)

---
updated-dependencies:
- dependency-name: "@types/jsdom"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4476 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/jsdom-21.1.5

dependabot[bot] committed:
Bump @types/cookie-parser from 1.4.5 to 1.4.6

Bumps [@types/cookie-parser](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/cookie-parser) from 1.4.5 to 1.4.6.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/cookie-parser)

---
updated-dependencies:
- dependency-name: "@types/cookie-parser"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4478 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/cookie-parser-1.4.6

dependabot[bot] committed:
Bump msw from 2.0.1 to 2.0.6

Bumps [msw](https://github.com/mswjs/msw) from 2.0.1 to 2.0.6.
- [Release notes](https://github.com/mswjs/msw/releases)
- [Changelog](https://github.com/mswjs/msw/blob/main/CHANGELOG.md)
- [Commits](https://github.com/mswjs/msw/compare/v2.0.1...v2.0.6)

---
updated-dependencies:
- dependency-name: msw
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4486 from Crown-Commercial-Service/dependabot/npm_and_yarn/msw-2.0.6

dependabot[bot] committed:
Bump @babel/core from 7.23.2 to 7.23.3

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.23.2 to 7.23.3.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.23.3/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4480 from Crown-Commercial-Service/dependabot/npm_and_yarn/babel/core-7.23.3

dependabot[bot] committed:
Bump @types/express-session from 1.17.7 to 1.17.10

Bumps [@types/express-session](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/express-session) from 1.17.7 to 1.17.10.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/express-session)

---
updated-dependencies:
- dependency-name: "@types/express-session"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4481 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/express-session-1.17.10

dependabot[bot] committed:
Bump @types/serve-favicon from 2.5.6 to 2.5.7

Bumps [@types/serve-favicon](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/serve-favicon) from 2.5.6 to 2.5.7.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/serve-favicon)

---
updated-dependencies:
- dependency-name: "@types/serve-favicon"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4479 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/serve-favicon-2.5.7

dependabot[bot] committed:
Bump @types/sinon-chai from 3.2.11 to 3.2.12

Bumps [@types/sinon-chai](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/sinon-chai) from 3.2.11 to 3.2.12.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/sinon-chai)

---
updated-dependencies:
- dependency-name: "@types/sinon-chai"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4485 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/sinon-chai-3.2.12

dependabot[bot] committed:
Bump express-fileupload and @types/express-fileupload

Bumps [express-fileupload](https://github.com/richardgirges/express-fileupload) and [@types/express-fileupload](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/express-fileupload). These dependencies needed to be updated together.

Updates `express-fileupload` from 1.4.0 to 1.4.2
- [Release notes](https://github.com/richardgirges/express-fileupload/releases)
- [Commits](https://github.com/richardgirges/express-fileupload/compare/v1.4.0...v1.4.2)

Updates `@types/express-fileupload` from 1.4.1 to 1.4.4
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/express-fileupload)

---
updated-dependencies:
- dependency-name: express-fileupload
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: "@types/express-fileupload"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4482 from Crown-Commercial-Service/dependabot/npm_and_yarn/express-fileupload-and-types/express-fileupload-1.4.2

dependabot[bot] committed:
Bump exceljs from 4.3.0 to 4.4.0

Bumps [exceljs](https://github.com/exceljs/exceljs) from 4.3.0 to 4.4.0.
- [Release notes](https://github.com/exceljs/exceljs/releases)
- [Commits](https://github.com/exceljs/exceljs/compare/v4.3.0...v4.4.0)

---
updated-dependencies:
- dependency-name: exceljs
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4483 from Crown-Commercial-Service/dependabot/npm_and_yarn/exceljs-4.4.0

dependabot[bot] committed:
Bump helmet from 7.0.0 to 7.1.0

Bumps [helmet](https://github.com/helmetjs/helmet) from 7.0.0 to 7.1.0.
- [Changelog](https://github.com/helmetjs/helmet/blob/main/CHANGELOG.md)
- [Commits](https://github.com/helmetjs/helmet/compare/v7.0.0...v7.1.0)

---
updated-dependencies:
- dependency-name: helmet
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4484 from Crown-Commercial-Service/dependabot/npm_and_yarn/helmet-7.1.0

NToppingCCS committed:
Fix test

NToppingCCS committed:
Merge pull request #4285 from Crown-Commercial-Service/feature/CAS-1281_Unified_AS

Point Buyer UI to unified Agreement Service

dependabot[bot] committed:
Bump @typescript-eslint/parser from 6.9.1 to 6.11.0

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 6.9.1 to 6.11.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v6.11.0/packages/parser)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/parser"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4487 from Crown-Commercial-Service/dependabot/npm_and_yarn/typescript-eslint/parser-6.11.0

dependabot[bot] committed:
Bump @types/require-directory from 2.1.4 to 2.1.5

Bumps [@types/require-directory](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/require-directory) from 2.1.4 to 2.1.5.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/require-directory)

---
updated-dependencies:
- dependency-name: "@types/require-directory"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4488 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/require-directory-2.1.5

dependabot[bot] committed:
Bump @types/config from 3.3.2 to 3.3.3

Bumps [@types/config](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/config) from 3.3.2 to 3.3.3.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/config)

---
updated-dependencies:
- dependency-name: "@types/config"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4492 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/config-3.3.3

dependabot[bot] committed:
Bump @types/cookies from 0.7.9 to 0.7.10

Bumps [@types/cookies](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/cookies) from 0.7.9 to 0.7.10.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/cookies)

---
updated-dependencies:
- dependency-name: "@types/cookies"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4490 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/cookies-0.7.10

dependabot[bot] committed:
Bump @types/nunjucks from 3.2.5 to 3.2.6

Bumps [@types/nunjucks](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/nunjucks) from 3.2.5 to 3.2.6.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/nunjucks)

---
updated-dependencies:
- dependency-name: "@types/nunjucks"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4491 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/nunjucks-3.2.6

dependabot[bot] committed:
Bump @types/node from 20.9.0 to 20.9.2

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 20.9.0 to 20.9.2.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4493 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/node-20.9.2

dependabot[bot] committed:
Bump prettier from 3.0.3 to 3.1.0

Bumps [prettier](https://github.com/prettier/prettier) from 3.0.3 to 3.1.0.
- [Release notes](https://github.com/prettier/prettier/releases)
- [Changelog](https://github.com/prettier/prettier/blob/main/CHANGELOG.md)
- [Commits](https://github.com/prettier/prettier/compare/3.0.3...3.1.0)

---
updated-dependencies:
- dependency-name: prettier
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4496 from Crown-Commercial-Service/dependabot/npm_and_yarn/prettier-3.1.0

dependabot[bot] committed:
Bump eslint from 8.53.0 to 8.54.0

Bumps [eslint](https://github.com/eslint/eslint) from 8.53.0 to 8.54.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v8.53.0...v8.54.0)

---
updated-dependencies:
- dependency-name: eslint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4494 from Crown-Commercial-Service/dependabot/npm_and_yarn/eslint-8.54.0

dependabot[bot] committed:
Bump @types/pa11y from 5.3.6 to 5.3.7

Bumps [@types/pa11y](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/pa11y) from 5.3.6 to 5.3.7.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/pa11y)

---
updated-dependencies:
- dependency-name: "@types/pa11y"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4495 from Crown-Commercial-Service/dependabot/npm_and_yarn/types/pa11y-5.3.7

dependabot[bot] committed:
Bump @typescript-eslint/eslint-plugin from 6.9.1 to 6.11.0

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 6.9.1 to 6.11.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v6.11.0/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #4489 from Crown-Commercial-Service/dependabot/npm_and_yarn/typescript-eslint/eslint-plugin-6.11.0
